Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -12001,6 +12026,313 @@ def pan_os_delete_redistribution_profile_command(args):
+
+    for entry in entries:
+        human_readable.append(
+            {
+                'Name': extract_objects_info_by_key(entry, '@name'),
+                'Description': extract_objects_info_by_key(entry, 'description'),
+                'Tags': extract_objects_info_by_key(entry, 'tag'),
+                'Source Zone': extract_objects_info_by_key(entry.get('from', {}), 'zone'),
+                'Source Interface': extract_objects_info_by_key(entry.get('from', {}), 'interface'),
+                'Source Address': extract_objects_info_by_key(entry, 'source'),
+                'Source User': extract_objects_info_by_key(entry, 'source-user'),
+                'Destination Address': extract_objects_info_by_key(entry, 'destination'),
+                'Action': list(entry['action'].keys())[0] if entry.get('action') else None
+            }
+        )
+
+        context.append(
+            {
+                'Name': extract_objects_info_by_key(entry, '@name'),
+                'Description': extract_objects_info_by_key(entry, 'description'),
+                'Tags': extract_objects_info_by_key(entry, 'tag'),
+                'SourceZone': extract_objects_info_by_key(entry.get('from', {}), 'zone'),
+                'SourceInterface': extract_objects_info_by_key(entry.get('from', {}), 'interface'),
+                'SourceAddress': extract_objects_info_by_key(entry, 'source'),
+                'SourceUser': extract_objects_info_by_key(entry, 'source-user'),
+                'DestinationAddress': extract_objects_info_by_key(entry, 'destination'),
+                'Action': entry.get('action'),
+                'EnforceSymmetricReturn': entry.get('enforce-symmetric-return'),
+                'Target': entry.get('target'),
+                'Application': extract_objects_info_by_key(entry, 'application'),
+                'Service': extract_objects_info_by_key(entry, 'service')
+            }
+        )
+
+    return human_readable, context
+
+
+def pan_os_list_pbf_rules_command(args):
+    name = args.get('rulename')
+    pre_post = args.get('pre_post')
+    show_uncommitted = argToBoolean(args.get('show_uncommitted', False))
+
+    raw_response = pan_os_list_pbf_rules(name=name, pre_post=pre_post, show_uncommitted=show_uncommitted)
+    result = raw_response.get('response', {}).get('result', {})
+
+    # the 'entry' key could be a single dict as well.
+    entries = (result.get('pbf', {}).get('rules', {}).get('entry')) or [result.get('entry')]
+    if not isinstance(entries, list):  # when only one nat rule is returned it could be returned as a dict.
+        entries = [entries]
+
+    if not name:
+        # filter the pbf-rules by limit - name means we get only a single entry anyway.
+        page = arg_to_number(args.get('page'))
+        if page is not None:
+            if page <= 0:
+                raise DemistoException(f'page {page} must be a positive number')
+            page_size = arg_to_number(args.get('page_size')) or 50
```

Comment:
for the whole file, we should extract 50 to be DEFAULT_PANORAMA_LIMIT or some better name

Reply:
you are right, ill change